src/webgl/p5.Shader.js

```diff
@@ -476,9 +476,12 @@ p5.Shader.prototype.setUniform = function(uniformName, data) {
       break;
     case gl.SAMPLER_2D:
       gl.activeTexture(gl.TEXTURE0 + uniform.samplerIndex);
-      uniform.texture =
-        data instanceof p5.Texture ? data : this._renderer.getTexture(data);
+      uniform.texture = data instanceof p5.Texture  ?
+        data : this._renderer.getTexture(data);
       gl.uniform1i(location, uniform.samplerIndex);
+      if (uniform.texture.src.gifProperties){
```

Comment:
Super minor, but would you mind adding a space after the closing bracket here, and using one space before the `?` on line 479 to match the formatting in the rest of the file?

Reply:
Okay, will do that. 
But I have a doubt, main branch of my fork was a few commits behind the `processing/p5.js:main` so I synced my fork, and did a git pull on my `main` branch, so now my `main` branch includes some code changes which aren't in the branch I created to do this PR. What should I do now ?

Reply:
I think either is OK! I believe Github is smart enough to not show content from those other commits in the diff, since that code is also on the branch that we're looking to merge this PR into.